src/libs/TextInputUtils/index.ios.js

```diff
@@ -0,0 +1,19 @@
+/**
+* Set value and selection of uncontrolled TextInput element
+*
+* @param {Object} textInputElement
+* @param {String} text
+* @param {Object} selection: {start: number, end: number}
+*/
+function setTextAndSelection(textInputElement, text, selection) {
```

Comment:
@stitesExpensify This function is implemented differently on Android|IOS|Web. What do you think about this imlementation?

Reply:
I think I'm missing some context. Why does this need to be implemented in 3 different ways @K4tsuki ?

Reply:
@stitesExpensify for the web, setNativeProps for `selection` doesn't work, because to set the selection of text input in web the property name is `selectionStart` and `selectionEnd`. So it has different name than native `selection` property.

For IOS, at first I am using android code, but  there is a bug, like what @parasharrajat mention above.
> There is an issue with IOS. Where after adding the emoji the cursor is moved to the start of the text.

The native behavior of setting selection using `setNativeProps` is different in IOS and android, so I am just adjusting the code based on native behavior of android and IOS.